src/main/java/org/mskcc/cbio/oncokb/domain/enumeration/MailType.java

```diff
@@ -16,6 +16,7 @@ public enum MailType {
     , CLARIFY_ACADEMIC_NON_INSTITUTE_EMAIL("clarifyAcademicUseWithoutInstituteEmail", "Clarify - Requested academic license from a non-institute email", null)
     , CLARIFY_USE_CASE("clarifyUseCase", "Clarify - Use case unclear", null)
     , CLARIFY_DUPLICATE_USER("clarifyDuplicateUser", "Clarify - User registered multiple accounts", null)
+    , CLARIFY_REGISTRATION_INFO("clarifyUserInfo", "Clarify - Detailed registration info required", null)
```

Comment:
I think the `templateName` should match with the filename `clarifyRegistrationInfo`.